actors/user/src/main/java/org/sunbird/user/actors/UserExternalIdManagementActor.java

```diff
@@ -185,6 +188,9 @@ private static void deleteUserExternalId(Map<String, String> map) {
     map.remove(JsonKey.ORIGINAL_EXTERNAL_ID);
     map.remove(JsonKey.ORIGINAL_ID_TYPE);
     map.remove(JsonKey.ORIGINAL_PROVIDER);
+    if (map.containsKey(JsonKey.EXTERNAL_ID)) {
```

Comment:
What if we dont do this while deleting? Given that all external id will become plain text, am not sure if this needed.
Also, did you really intend to put it inside the map or remove (all previous lines seem to be removing)?

Reply:
Decryption is not required here , becuase data will be in plain text inside DB. SO just now i remove this check.